messages/cache_setting/errors.go

```diff
@@ -1,4 +1,4 @@
-package cache_settings
+package cachesetting
```

Comment:
Just a small doubt. 
But why remove the `_` from the name? 
I can see that `edge_applications` is using it, so shouldn't this use it also? 
To keep pattern between names on the project i mean.

Reply:
I think we can remove the underscore from edge_applications then :) 
I feel like this one makes more sense for a package name in go

Reply:
That's great, lets keep the pattern! 
You know the best practices in GO and i am trying to keep up. 👍

Reply:
@helio-neto I changed it because they are good practice and when you go to import it automatically works the short way and with underline as @PatrickMenoti commented